integration_tests/client/datatype/test_data_generation.py

```diff
@@ -404,6 +404,7 @@ def test_generate_prediction_data(client: Client):
         },
         "parameters": {
             "task_type": TaskType.OBJECT_DETECTION.value,
+            "metric_params": None,
```

Comment:
note: I like this `metric_params` concept. if Charles/Eric like this idea, we could move a bunch of our other parameters under this umbrella.